src/Microsoft.DotNet.Tools.Compiler.Native/project.json

```diff
@@ -25,7 +25,8 @@
         },
       "Microsoft.DotNet.ILCompiler": "1.0.1-*",
       "Microsoft.DotNet.ObjectWriter": "1.0.2-*",
-      "Microsoft.DotNet.RyuJit": "1.0.0-*"
+      "Microsoft.DotNet.RyuJit": "1.0.0-*",
```

Comment:
Ideally, the dependency on ObjectWriter and RyuJit should be abstracted with ILCompiler's dependencies and not be exposed to dotnet-compile-native.

And dotnet-compile-native should pick the latest ILCompiler from the feed assuming ILC won't break the command line options contract.

@brthor, what do you think about this?